pset-functions.js

```diff
@@ -30,7 +30,26 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 1:')
 
 // Add your code below this line
-
+function blackJack(playerCardScore, dealerCardScore){
```

Comment:
@cbrooks2019 nice approach to leverage the order of your `if..else if` statements to handle all of the potential scenarios. Check out the solution for an alternative branching approach to the `if..else if` clause for this problem: https://github.com/jsd20191008/hw-01-functions-solution/blob/master/pset-functions.js#L33